docs/manage/reference/viam-agent/_index.md

```diff
@@ -174,158 +208,120 @@ Edit and fill in the attributes as applicable.
 {{% /tab %}}
 {{< /tabs >}}
 
-### `viam-agent`
+## `version_control`: Version management for `viam-agent` and `viam-server`
+
+By default, when a new version of `viam-server` becomes available, `viam-agent` will restart and upgrade `viam-server` immediately.
```

Comment:
I believe this is not accurate. agent will download the new version when available but it will not be used until agent restarts. This is why we introduce maintenance windows so the agent knows when is safe to do it automatically.

Reply:
line 227 is accurate

Reply:
Correct, either restarts viam-server or power cycles